packages/backend/src/rapyd/utills/request.ts

```diff
@@ -0,0 +1,101 @@
+import axios from 'axios'
+import crypto from 'crypto'
+import env from '../../config/env'
+
+const generateRandomString = (size: number) => {
+  try {
+    return crypto.randomBytes(size).toString('hex')
+  } catch (error) {
+    console.error('Error generating salt')
```

Comment:
I think you should use the logger
I don't see any reasons to keep the console logs if we have the logger now